src/context/TxContext/index.tsx

```diff
@@ -102,63 +82,76 @@ function TxContextProvider({children}: PropTypes) {
             .filter(({event: {section, method}}) => section === 'system' && method === 'ExtrinsicFailed')
             // we know that data for system.ExtrinsicFailed is
             // (DispatchError, DispatchInfo)
-            .map(
-              ({
+            .map((eventRecord) => {
+              const {
                 event: {
                   data: [error],
                 },
-              }) => {
-                if ((error as any).isModule) {
-                  // for module errors, we have the section indexed, lookup
-                  const decoded = api.registry.findMetaError((error as any).asModule);
-                  const {documentation} = decoded;
-
-                  return documentation.join(' ').trim();
-                } else {
-                  // Other, CannotLookup, BadOrigin, no extra info
-                  return error.toString();
-                }
-              },
-            );
+              } = eventRecord;
+
+              if ((error as any).isModule) {
```

Comment:
why `any`? There should be a type for this error

Reply:
yes you are right, I found an example and did the same 
https://github.com/polkadot-js/apps/blob/d1389052c43c24585ce87b17da8a82dc6e8294e9/packages/react-components/src/Status/Queue.tsx#L88